test/integration/agent/gobgp_test.go

```diff
@@ -50,74 +48,50 @@ func TestGoBGPLifecycle(t *testing.T) {
 		RouterID:   routerID2,
 		ListenPort: listenPort2,
 	}
-	server3GlobalConfig := &bgp.GlobalConfig{
-		ASN:        uint32(asn3),
-		RouterID:   routerID3,
-		ListenPort: listenPort3,
-	}
+
+	originalLogLevel := log.GetCurrentLogLevel()
+	require.NoError(t, log.SetLogLevel("4"))
+	defer func() {
+		log.SetLogLevel(originalLogLevel)
+	}()
```

Comment:
I don't think this is available without calling `log.InitLogs`, hence the test failure

Reply:
You are right. However, it is a little complicated to use `log.InitLogs` here. As a result, I set `klog` level to `4`, which is `debug`, directly. Unfortunately, I didn't find a method to get the current log level of `klog`. We can only assume that the default log level `0` is used.

One more thing, the change is global, it can affect the other tests logging if we run the integration test in parallel. Fortunately, we run the integration test in single process. The log level of debug should only happen within the current test `TestGoBGPLifecycle`.